tests/eth_call_test.go

```diff
@@ -1,26 +1,22 @@
 package tests
 
 import (
-	"fmt"
 	"math"
 	"testing"
 
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/stretchr/testify/require"
 )
 
-func TestEthCall_CodeLargerThanMaxInitCodeSizeIsNotAccepted(t *testing.T) {
+func TestEthCall_CodeLargerThanMaxInitCodeSizeIsAccepted(t *testing.T) {
 	tests := map[string]struct {
 		codeSize int
-		err      error
 	}{
-		"max code size": {
+		"max code size lfvm": {
```

Comment:
Can you also add two more test cases for a code that has `params.MaxCodeSize` and `params.MaxCodeSize+1`?

The way it is now, both cases are covered by geth.